added CVS vending machine

DavidHunterJS opened the pull request:
I've added the CVS vending machine using the recommend code form bhousel  in #2894 as well as these tags

`payment: yes`
`weelchair: yes`
`covered: yes`

taken from [this press release](https://cvshealth.com/newsroom/press-releases/cvs-pharmacy-thinks-outside-box-introduction-health-and-wellness-vending) from CVS Health about the machines.

bhousel commented:
Thanks and welcome @DavidHunterJS 🎉 

I'm not sure how I feel about the other tags `payment` `wheelchair` and `covered` - we don't use these tags anywhere else in the index and so I'd prefer for mappers to supply them.  (Anything that appears in this index should be considered "universally true" for the brands).

I removed the extra tags and resolved the merge conflicts in 2cb700d1f

I also changed the names of all the CVS entries to have `"name": "CVS Pharmacy"` and `"short_name": "CVS"`.  This is something someone mentioned on different ticket a while ago #2236, but we never got around to doing it.  (Reasoning for this decision is because the storefronts are signed as "CVS Pharmacy" and it is what the brand calls itself, so that can be the value for the `name`.  However it's totally ok to use the "CVS" to label these features on a map, so that can be a `short_name`).

DavidHunterJS commented:
I get the universality aspect of that, thanks for the update!

On Sat, Jul 20, 2019 at 11:46 AM Bryan Housel <notifications@github.com>
wrote:

> Thanks and welcome @DavidHunterJS <https://github.com/DavidHunterJS> 🎉
>
> I'm not sure how I feel about the other tags payment wheelchair and
> covered - we don't use these tags anywhere else in the index and so I'd
> prefer for mappers to supply them. (Anything that appears in this index
> should be considered "universally true" for the brands).
>
> I removed the extra tags and resolved the merge conflicts in 2cb700d
> <https://github.com/osmlab/name-suggestion-index/commit/2cb700d1f8a79a40d0a1c2181191fade6700701d>
>
> I also changed the names of all the CVS entries to have "name": "CVS
> Pharmacy" and "short_name": "CVS". This is something someone mentioned on
> different ticket a while ago #2236
> <https://github.com/osmlab/name-suggestion-index/issues/2236>, but we
> never got around to doing it. (Reasoning for this decision is because the
> storefronts are signed as "CVS Pharmacy" and it is what the brand calls
> itself, so that can be the value for the name. However it's totally ok to
> use the "CVS" to label these features on a map, so that can be a
> short_name).
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/osmlab/name-suggestion-index/pull/2905?email_source=notifications&email_token=ACURP2INJ3UZZV4QXRZU3DDQANMQLA5CNFSM4IFNAL3KYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD2NT4VA#issuecomment-513490516>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACURP2KS2CH5OGOXQVKCHNLQANMQLANCNFSM4IFNAL3A>
> .
>
